compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -199,6 +288,9 @@ static LogicalResult setRootConfigImpl(func::FuncOp entryPointFn, Operation *op,
         // TODO (nmeshram): This is very limited for now, plan is to
         // let it first crash for all the other ops and then consiously
         // add support for them, this way we can verify our work.
+        .Case<linalg::GenericOp>([&](auto op) {
```

Comment:
The switch cases need to be ordered from most specific to most general. If we move this `case` to a `LinalgOp` then this needs to go after the `case` for `linalg.matmul`.